guides/v2.3/graphql/reference/quote.md

```diff
@@ -137,6 +118,9 @@ The following returns information about a cart given a `cart_id`. Note that the
 
 ## Mutations
 
+### Create an empty cart {#createEmptyCart}
+{:.no_toc}
```

Comment:
This heading should appear in the right nav

Reply:
removed text